.storybook/main.js

```diff
@@ -0,0 +1,37 @@
+const path = require("path");
+const tsconfigPathsPlugin = require("tsconfig-paths-webpack-plugin");
+
+module.exports = {
+  stories: ["../src/**/*.stories.mdx", "../src/**/*.stories.@(js|jsx|ts|tsx)"],
+  addons: [
+    "@storybook/addon-a11y",
+    "@storybook/addon-essentials",
+    "@storybook/addon-interactions",
+    "@storybook/addon-links",
+  ],
+  framework: "@storybook/react",
+  core: {
+    builder: "@storybook/builder-webpack5",
+  },
+  webpackFinal(config) {
+    config.resolve.modules = [...(config.resolve.modules || []), path.resolve(__dirname, "../src")];
+    config.resolve.plugins = [...(config.resolve.plugins || []), new tsconfigPathsPlugin()];
+    config.module.rules.push({
+      test: /\.scss$/,
+      use: [
+        "style-loader",
+        {
+          loader: "css-loader",
+          options: {
+            importLoaders: 1, // 1 => postcss-loader
+            modules: {
+              localIdentName: "[local]___[hash:base64:2]",
+            },
+          },
+        },
+        "sass-loader",
+      ],
+    });
```

Comment:
storybook内でパスエイリアス使うためにこれ必要じゃないですか？

```suggestion
    });
    config.resolve.alias = {
      ...config.resolve.alias,
      '~': path.resolve(__dirname, '../src'),
    };
```

Reply:
あんまりプラグインに頼るなって言われたらそうなんだけど、
https://github.com/shikosai32/shikosai.net/pull/17/files#diff-cafe2123a72c4ce3a9f7e9ee4b0e188256eb02d7ec1e54fa2cadfac99b22f92bR17-R18
ここで対応してます

https://github.com/shikosai32/shikosai.net/pull/17/files#diff-98b614e1838b171ee71c04450a8f1a562753193fb7d4f53a4de8b9b5a7e980eeR2
ここで使ってる（storybookのビルドは通っているはず）

Reply:
なるほど、じゃぁ大丈夫ですね。
あｐｐろヴぇ